modules/serverless-rn-1-2-0.adoc

```diff
@@ -0,0 +1,42 @@
+* Access to the KnativeServing resource is now restricted to `cluster-admin` roles to prevent any user from blocking the resource. Only `cluster-admin` roles can create KnativeServing CRs.
+* The {ServerlessProductName} Operator can now be found in the OperatorHub by searching for "knative".
+* The {product-name} web console now shows status conditions for the KnativeServing resource.
+* In version 1.2.0, the {ServerlessProductName} Operator inspects network policies for namespaces.
++
+If no network policy exists, the Operator will automatically create a wide open policy, to ensure that traffic can flow in and out of the namespace and OpenShift routes can be used.
++
+If there is an existing network policy, {ServerlessProductName} will not create a new policy. The Operator will expect the user to continue managing their own network policies as needed for their applications. For example, the user will need to set policies that allow traffic to flow in and out of the namespace, and allow OpenShift routes to still be used after the namespace is added to a ServiceMeshMemberRoll.
+
+
+[id="fixed-issues-1-2-0_{context}"]
+== Fixed issues
+* In previous releases, using the same services or routes in different namespaces caused services to not work properly and caused {product-title} routes to be overriden. This issue has been fixed.
+* In previous releases, different traffic split targets required a mandatory tag. A single traffic split can now be defined with `untagged` traffic targets.
+* Existing Knative Services and Routes which had been created with public visibility in {ServerlessProductName} Operator version 1.1.0 were not able to be updated to cluster-local visibility. This issue is now fixed.
+* `Unknown Uninitialized : Waiting for VirtualService` error has been fixed.
+* Knative service no longer returns a 503 status code when the cluster is running for a
+long time.
+
+[id="known-issues-1-2-0_{context}"]
+== Known issues
+* Installing the {ServerlessProductName} Operator on {product-title} versions older than 4.2.4 using OLM may incorrectly use community versions of the required dependencies. As a workaround, on  {product-title} versions older than 4.2.4, explicitly install the Red Hat provided versions of the Elastic Search, Jaeger, Kiali and ServiceMesh Operators before installing the {ServerlessProductName} Operator.
+* If you are upgrading the {ServerlessProductName} from version 1.1.0 to version 1.2.0 and you have set up a ServiceMeshControlPlane and ServiceMeshMemberRoll to work with your Knative Serving instance, you will need to remove the `knative-serving` namespace and any other namespaces that contain Knative Services from the ServiceMeshMemberRoll in `istio-system`.
++
+You can also delete the ServiceMeshControlPlane from the namespace entirely if it is not required for other applications.
```

Comment:
Let's add a sentence or two here on what interruptions are expected from the upgrade (as requested by Paul):

> Once the upgrade starts, existing services will continue to work as before. Once you unblock the release by removing the `knative-serving` (and all other relevant) namespaces from the ServiceMeshMemberRoll there will be a brief outage to all active services. This will fix itself. Make sure to remove all namespaces containing Knative Services from the original ServiceMeshMemberRoll.